ONSdigital/census-rm-ops-ui
# View All Case Details

# Motivation and Context
Offers user ability to view all case and event details of selected case

# What has changed
New page to view all case details
New templates
Tests

# How to test?
Build case-api image and run ops-ui locally. Select a case and check you can see the details
Load sample file locally and go to http://localhost:8234/

# Links
https://trello.com/c/ZxxGq4aT/1222-r-ops-ui-view-all-case-details-13
https://rops-prototypes.netlify.app/prototypes/rops-management/version-2/1-acacia-avenue/index.html
